plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -979,7 +979,7 @@ public static Object of(
     @Override
     public void testShowCreateTable()
     {
-        assertQueryFails("SHOW CREATE TABLE region", "No PropertyMetadata for property: pinotColumnName");
```

Comment:
So we have generic tests to prevent bugs like https://github.com/trinodb/trino/issues/14071, but then we disable tests because we have bugs. 
What can we improve in our process to avoid having bugs lingering in the code?

Reply:
This test class was migrated to BaseConnectorSmokeTest from AbstractTestQueryFramework 6 days ago in #13619. The bug was found and the test was disabled during the migration, so it's not "lingering" in my opinion. 

(It's not answer to "What can we improve in our process to avoid having bugs lingering in the code?" though)

Reply:
It'd be better if the code here had a TODO comment and a github issue link.

Anyway, thanks for explaining. It's good it was not lingering.